src/mudlet-mapper.xml

```diff
@@ -3186,58 +3769,131 @@ end
 
 if not area then mmp.echo(matches[3]..&quot; isn't a known area. Which one do you want to set?&quot;) return end
 
-local tmp = getRoomUserData(1, &quot;areaContinents&quot;)
```

Comment:
Given how the script works to ensure maximum compatibility with most versions of Mudlet, using map userdata won't happen that soon.